packages/inter-protocol/src/vaultFactory/vault.js

```diff
@@ -766,22 +768,43 @@ const selfBehavior = {
    *
    * @see getActualDebAmount
    *
-   * @param {MethodContext} context
    * @returns {import('./storeUtils.js').NormalizedDebt} as if the vault was open at the launch of this manager, before any interest accrued
    */
-  getNormalizedDebt: ({ state }) => {
+  getNormalizedDebt() {
+    const { state } = this;
     // @ts-expect-error cast
     return reverseInterest(state.debtSnapshot, state.interestSnapshot);
   },
 };
 
-const makeVaultBase = defineDurableKindMulti(
-  makeKindHandle('Vault'),
-  initState,
+export const VaultI = M.interface('Vault', {
+  getCollateralAmount: M.call().returns(AmountShape),
+  getCurrentDebt: M.call().returns(AmountShape),
+  getNormalizedDebt: M.call().returns(AmountShape),
+  getVaultSeat: M.call().returns(SeatShape),
+  initVaultKit: M.call(SeatShape, M.any()).returns(M.promise()),
```

Comment:
`M.any()`? What's that for? `storageNode`... hm... `M.remotable()` instead? or `M.eref(M.remotable())`?